part-spec/power.json

```diff
@@ -697,6 +697,516 @@
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTrickleChargeCurrentTyp": {
+          "description": "typical charging current of a device in trickle charge phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTrickleChargeCurrentMin": {
+          "description": "minimum charging current of a device in trickle charge phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTrickleChargeCurrentMax": {
+          "description": "maximum charging current of a device in trickle charge phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTrickleChargeCurrentAccuracyTyp": {
+          "description": "typical Trickle charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTrickleChargeCurrentAccuracyMin": {
+          "description": "minimum Trickle charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTrickleChargeCurrentAccuracyMax": {
+          "description": "maximum Trickle charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentTyp": {
+          "description": "typical charging current of a device in charge termination phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentMin": {
+          "description": "minimum charging current of a device in charge termination phase",
+          "comment": "units of volts",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentMax": {
+          "description": "maximum charging current of a device in charge termination phase",
+          "comment": "units of amps",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentAccuracyTyp": {
+          "description": "typical termination charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentAccuracyMin": {
+          "description": "minimum termination charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryTerminationChargeCurrentAccuracyMax": {
+          "description": "maximum termination charging current regulation accuracy of a device",
+          "comment": "units of %",
+          "$ref": "definitions.json#/unit"
+        },
+        "batteryChargeVoltageTyp": {
```

Comment:
couldn't this be variable - does a typical value make sense?

Reply:
Let's discuss more. I've mostly seen this defined as a value.